charts/camunda-platform/charts/zeebe-gateway/templates/gateway-deployment.yaml

```diff
@@ -25,12 +25,12 @@ spec:
       {{- end }}
       imagePullSecrets:
         {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}
-      {{- if .Values.extraInitContainers }}
       initContainers:
-        {{- with .Values.extraInitContainers }}
-          {{- tpl (toYaml . ) $ | nindent 8 }}
+        {{- if .Values.initContainers }}
```

Comment:
No need for this if statement, it's already defaulted to an empty list in the next line.